src/Actions/actions.js

```diff
@@ -26,6 +26,13 @@ function toggleModal_post(toggle) {
     }
 }
 
+function toggleModal_signIn(toggle) {
```

Comment:
It seems you have multiple TOGGLE_MODAL actions. Can you just combine them all as well?
```
type: 'TOGGLE_MODAL',
data: { modalType: X, isOn: true }
```